modules/ting_new_materials/ting_new_materials.module

```diff
@@ -167,7 +167,10 @@ function ting_new_materials_do_search($query, $page = 0, $num_per_page = 10) {
   // Search the data well.
   $search_result = array();
   try {
-    $search_result = ting_do_search($query, $page + 1, $num_per_page, $options);
+    // TODO: Consider porting this to the Search Abstraction Layer.
```

Comment:
yes please :-)

Reply:
We may need to look into this now as it blocks the removal of other important todos.